plugins/lookup/passwordstore.py

```diff
@@ -14,6 +14,8 @@
     description:
       - Enables Ansible to retrieve, create or update passwords from the passwordstore.org pass utility.
         It also retrieves YAML style keys stored as multilines in the passwordfile.
+      - To avoid problems when accesing multiple secrets at once, add ``auto-expand-secmem`` to
+        ``~/.gnupg/gpg-agent.conf``. Where this isn't possible, consider using ``lock=readwrite`` instead.
```

Comment:
```suggestion
      - To avoid problems when accesing multiple secrets at once, add C(auto-expand-secmem) to
        C(~/.gnupg/gpg-agent.conf). Where this isn't possible, consider using C(lock=readwrite) instead.
```